### PR TITLE
Delete preset

### PR DESCRIPTION
**Pull request by jochem725**
## Today I present: Preset deletion!
Please verify everything works correctly since I had some issues with preset updating.

### CODE REVIEW

File: frontend/public/js/preset.js [hunk 1]
```diff
@@ -49,7 +49,14 @@ Preset.prototype = {
 		switchCurrentView(this.cameraid);
 		console.log("Recall preset id: " + this.id);
 	},
-	
+
+	/**
+	 * Recall this preset.
```
Comment (bvanwijk) [inline frontend/public/js/preset.js]: I think you mean delete this preset

**Comment by bvanwijk** [comment]
Except from the little comment I made, I think it looks good.
Everything seems to be working correctly.

**Comment by jochem725** [comment]
@bvanwijk You are right, thanks for pointing it out. I've fixed it.

**Comment by jochem725** [comment]
I also forgot to add test cases, I've resolved this as well.

**Comment by egedebruin** [comment]
Looks good, it works fine here so I agree with merging.

**Comment by bvanwijk** [comment]
Looks great,  think you can merge this now.

**Comment by NaomidR** [comment]
Looks good now, I agree with merging.

**Pull request merged**